packages/pluggableWidgets/tree-view-web/src/components/__tests__/TreeView.spec.tsx

```diff
@@ -0,0 +1,102 @@
+import { GUID } from "mendix";
+import { createElement } from "react";
+import { mount } from "enzyme";
+import { TreeView, TreeViewProps } from "../TreeView";
+
+const items: TreeViewProps["items"] = [
+    { id: "11" as GUID, value: "First header", content: <div>First content</div> },
```

Comment:
We have a builder for it inside piw-internal :)

Reply:
Oh I didn't know about those, thanks 😄  But in this case I do not need a `List(Attribute)Value`, but only the id as a `GUID` since in this component level I already mapped the objects from Mendix into my own format. Using the builder will not add much value, so I will keep it like this for now 😄